<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserMode</class>
 <widget class="QWidget" name="UserMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserMode</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListView" name="dogsList"/>
   </item>
   <item>
    <widget class="QPushButton" name="addButton">
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="adoptionList"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
